### DISCUSSION TITLE
Date Picker and Async Combobox

### DISCUSSION THREAD

**Discussion by fortezhuo** [rewindui/rewindui#83] [General]
Hi,

Thanks for great library.

How to create datepicker and Async Combobox component ?

**Comment by NickDunas** [comment]
Hello @fortezhuo!

Datepicker component is on my roadmap, but it's not yet implemented. You can always use the Input component with type="date" though.

Regarding async Combobox please take a look here: https://rewind-ui.dev/components/combobox#on-search

You will have to add the onSearch event to get the input from the search field and filter out the combobox options based on your async call.

**Comment by fortezhuo** [comment]
Hi, Thanks for the quick response.

Actually I have my own TailiwindCSS UI library, but it not maintained well and lacks of unit testing. If you want, I want to rewrite my library using rewindui code standard, and port the components into this repo. 

Mine using this following libraries :
- DatePicker : @rehookify/datepicker
- Uploader : react-dropzone
- WYSIWYG Editor : @tiptap/react
- Select : downshift
- Number (with thoudsand separator, decimal etc) : react-number-format  

And do you have any preference 3rd party for datepicker, uploader, etc ?

**Comment by NickDunas** [comment]
I really thank you for offering to port your components to Rewind-UI!

I'm keeping an eye on the bundle size, so I try to avoid using external libraries when I can. Right now, I'm using 'floating-ui' for all the floating components like Tooltip, Popover, Dropdown and so on. 'CVA' helps with styling, and I've only used 'framer-motion' for animating the Toast component (for other animations, I've used the Web Animations API).

I prefer not to rely on additional libraries, so if you decide to port your components, just keep in mind that we'll need to build everything from scratch, except for the floating elements.

I hope this doesn't discourage you!
